lib/index.js

```diff
@@ -112,13 +112,16 @@ exports.balm = (opts = {}) => {
     _changes = [...new Set(_changes)];
 
     _timer = setTimeout(() => {
+      if (!_changes.length) return;
```

Comment:
I'm wondering why I even had a _changes.length check if notifyClient only fires when something changed. Can you think through whether we even need it?

Reply:
maybe if there is a scenario if the timeout is running and another timeout gets registered while the `.send()` is happening? If that is even possible. That's only thing I can think of.